dts/bindings/display/solomon,ssd1306fb-common.yaml

```diff
@@ -60,3 +60,13 @@ properties:
       Time it takes for the device from power up to become responsive and
       accepting commands. Defaults to 10ms (found by trial and error) if not
       provided.
+
+  use-internal-iref:
+    type: boolean
+    description: |
+      Use internal Iref, not common but sometime used to save components cost
+
+  use-alternative-write:
```

Comment:
why cant this be kconfig

Reply:
what's the rationale, why can't the other dts properties be kconfig?

Reply:
I'm confused about your comment here:

> The write command for ssd1306 doesn't take into account the `segment_offset` DTS property. However the one used for ht1306 does. Allow
users to use that write method on non-ht1306 controllers.

Could we instead update the ssd1306 write logic to account for the `segment_offset` property?

Reply:
There are two write functions in the ssd1306 driver. 
Currently, which to use is decided based on the `compatible =` dts property. If it's `ssd1306`, the first one is used, it's `ht1306`, then the second one is used.

The particular display that the board in the linked PR has is an ssd1306, but has a segment offset that needs to be set to display the image correctly.

The second write method (originally intended for the ht1306, but that's not what we have) is the only one that works with this display.

We didn't want to touch the other write method as we don't have other hardware than this, and can't test that we don't introduce a regression.

Reply:
I didn't realize I did this, but I also commented with @narvalotech which is my personal account. Sorry bout that :)

Reply:
Please put more info in the description of the property in the binding so thats users of the binding know when or not to use it.

Reply:
We updated the `ssd1306_write_default` instead.

Reply:
I'm concerned this would introduce a regression as well, since it is only tested on this display. Could this display use the `ht1306` compatible? I'm not clear what the differences between these displays are. Otherwise perhaps we can test this with another display to check for regressions, @henrikbrixandersen do you have access to a display using this driver? I see you added it to a board in #70645

Reply:
It uses at least one command that is not listed in the `ht1306` command set / datasheet. I think it's the Iref command.

Reply:
Ok- from the datasheet it does seem like the offset should be used there- and no in tree display definitions for the SSD1306 that I can find are setting the segment offset to a nonzero value- so I think this change is ok.